src/components/Equity/EquityRow.tsx

```diff
@@ -103,20 +103,35 @@ export const EquityRow: React.FC<EquityRowProps> = ({
       <Flex flex={1} alignItems='flex-start' justifyContent='space-between' gap={4}>
         {typeof icon === 'string' ? <LazyLoadAvatar src={icon} /> : icon}
         <Flex flexDir='column' flex={1} gap={1} textAlign='left'>
-          <Stack
-            direction='row'
-            display={displayMdFlex}
-            gap={1}
-            color='chakra-body-text'
-            fontSize={fontSizeMdSm}
-            lineHeight='shorter'
-            divider={divider}
-          >
-            {labelJoined}
-          </Stack>
-          <RawText color='chakra-body-text' fontSize={fontSizeMdMd} display={displayMdNone}>
-            {label}
-          </RawText>
+          <Flex flex={1} alignItems='center' gap={2}>
+            <Stack
+              direction='row'
+              display={displayMdFlex}
+              gap={1}
+              color='chakra-body-text'
+              fontSize={fontSizeMdSm}
+              lineHeight='shorter'
+            >
+              <RawText>{label}</RawText>
+              {subText && divider}
+              {subText && <RawText>{subText}</RawText>}
```

Comment:
Removed `labelJoined` and moved this in-JSX to avoid having a divider between the subtext and the tag, which would look odd